lib/tasks/stt-task.js

```diff
@@ -44,6 +44,9 @@ class SttTask extends Task {
       }
     } else {
       this.data.recognizer = {hints: [], altLanguages: []};
+      // If there is no verb's recognizer, use application's recognizer for label
```

Comment:
hmm, whoa.....default to a label named 'default' ?  Why is this necessary?  What if the user has a label actually named 'default'?

Reply:
the logic later will check if label === default, then use cs.syntheizerLabel

We need to have a condition to identify when to use application default label.

Reply:
By using this special string: {jambonz_label}

If there is no verb synthesizer/recognizer, jambonz will replace this special string to application synthesizer/recognizer label.